Vector.hpp

```diff
@@ -177,7 +159,7 @@ class Vector
 	 *
 	 * @return The current number of elements in the vector.
 	 */
-	size_t Size() const
+	size_t size() const
```

Comment:
function names should describe an action `getSize()` is a better name.  functions like this with good names don't need docs at all